HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,711 @@
+2. As an hts-token-airdrop-sender,  I want to airdrop a token using to a receiver who has not associated with the token but has an open autoAssociation slot. I expect this should work exactly like the cryptoTransfer.
+3. As an hts-token-airdrop-sender,  I want to airdrop a token to a receiver who has not associated with the token and has no open autoAssociation slots. I expect this transfer will be marked as pending that the receiver can claim at any time if it is still available at that time.
+4. As an hts-token-airdrop-sender I want to cover the custom fees (fixed, fractional, and royalty excluding fallback) of an airdrop to an account that is already associated with a token to reduce the friction by a receiving account
+5. As an hts-token-airdrop-sender I want to cover the fees (custom, association & rent until the end of the first full auto renewal period) of an airdrop that consumes an automatic token slot to reduce the friction for a receiving account
+6. As an hts-token-airdrop-sender I want to cover the fees (airdrop, custom, association & rent until the end of the first full auto renewal period) of an airdrop that results in a pending transfer to reduce the friction by the intended receiving account.
+7. As an airdrop-sender I want to cancel pending token transfers that I initiated and are still unclaimed by the receiver.
+8. As a token treasury account holder, I want to airdrop tokens with custom fallback fees
+
+#### Receiver
+
+1. As a receiver, I want to be able to query to see if I have any pending airdrops targeted for my account.
+2. As a receiver who has pending airdrops, I want to be able to claim the airdrops using a Claim API. I expect to receive this airdrop without paying for association fee and custom fee. I understand that if the sender does not have this token in their account when I claim it or if they have cancelled the pending airdrop, this claim request will fail, and the pending airdrop will be deleted.
+3. As a receiver who claims a token airdrop I will only be responsible for the transaction submission fee. No custom fees will be assessed to me as part of this transfer. 
+4. As a user with receiver signature required enabled any airdrops sent to me will result in a pending transfer, even with open slots.
+
+### General User Stories
+
+1. As an hts-token-holder I want to be able to reject unwanted tokens (regardless of how they were obtained) by sending them back to the treasury account and not having to pay any custom fees.
+2. As a non-hedera-centric wallet user, I want to obtain an auto-created account (by EVM address or public key) that is able to receive unsolicited tokens. I understand this account will have `maxAutomaticAssociations` set to -1 implying unlimited automatic associations.
+3. As account owner for an account created by an EVM tool my experience with airdrops on Hedera is identical to that on Ethereum
```

Comment:
There is no "claiming of airdrops" on Ethereum, what does that US mean ?

Reply:
The assumption is that if you're using MetaMask / Brave / etc. then you have auto-associations set on your account. The only way your account could have been created was by someone sending value to your account, and since your account didn't exist before it is auto-created, and therefore has max associations set to -1, and thus your experience with airdrops will be the same as on other chains.

If a MetaMask / Brave / etc. user gets an account that was not auto-created or was not created with -1 as max associations, then they won't understand why they're not receiving airdrops. We'll need to make sure google/AI searches link to docs that lead them to hashscan where they can change to -1. We're of the belief that the number of such people will be a vanishingly small percentage, and with the MetaMask snap, we can actually handle this better / more directly. So while it isn't perfect, it should cover a very high percentage (well above 99% I would think).

Reply:
Additionally, though Ethereum doesn't have it on the network many flows in web3 do support a similar concept.
For example an account holding token A may be awarded Token B but only if they go through a series of steps such as going to a DApp and issuing a mint of the token.
A parallel UX could be applied here.